RecoBTag/TensorFlow/python/pfMassIndependentDeepDoubleXJetTags_cff.py

```diff
@@ -0,0 +1,15 @@
+import FWCore.ParameterSet.Config as cms
+from pfDeepDoubleBvLJetTags_cfi import pfDeepDoubleBvLJetTags
+from pfDeepDoubleCvLJetTags_cfi import pfDeepDoubleCvLJetTags
+from pfDeepDoubleCvBJetTags_cfi import pfDeepDoubleCvBJetTags
+
+pfMassIndependentDeepDoubleBvLJetTags = pfDeepDoubleBvLJetTags.clone(
+    graph_path = cms.FileInPath('RecoBTag/Combined/data/DeepDoubleX/94X/V01/DDB_mass_independent.pb')
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.

Reply:
what do you mean in this file? The only param set here is the model, which is unavoidable.

Reply:
"please drop type specifications for all parameters which already exist." means to drop e.g. ```cms.FileInPath``` type specification to get
```graph_path = 'RecoBTag/Combined/data/DeepDoubleX/94X/V01/DDB_mass_independent.pb'```

Reply:
this is still not resolved.